src/main/util.cpp

```diff
@@ -323,6 +331,14 @@ void cvc4_init() throw(Exception) {
 
 #endif /* __WIN32__ */
 
+  struct sigaction act5;
```

Comment:
This block should be in the `__WIN32__` ifndef above.

Reply:
Done.